beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -1373,12 +1379,18 @@ impl<AppReqId: ReqId, E: EthSpec> Network<AppReqId, E> {
         let peer_id = event.peer_id;
 
         if !self.peer_manager().is_connected(&peer_id) {
-            debug!(
-                self.log,
-                "Ignoring rpc message of disconnecting peer";
-                event
-            );
-            return None;
+            // Sync expectes a RPCError::Disconnected to drop associated lookups with this peer.
```

Comment:
```suggestion
            // Sync expects a RPCError::Disconnected to drop associated lookups with this peer.
```
I can has airdrop? ;)